test/infrastructure/inmemory/internal/controllers/inmemorymachine_controller.go

```diff
@@ -236,8 +236,8 @@ func (r *InMemoryMachineReconciler) reconcileNormal(ctx context.Context, cluster
 }
 
 func (r *InMemoryMachineReconciler) reconcileNormalCloudMachine(ctx context.Context, cluster *clusterv1.Cluster, _ *clusterv1.Machine, inMemoryMachine *infrav1.InMemoryMachine) (ctrl.Result, error) {
-	// Compute the resource group unique name.
-	// NOTE: We are using reconcilerGroup also as a name for the listener for sake of simplicity.
+	// Compute the name for resource group and listener.
+	// NOTE: we are using the same name for convenience, but it is not required.
```

Comment:
Not sure if it's clear here that same is referring to listener as we don't calculate the listenerName here

Reply:
Same further down